pipelinewise/fastsync/partialsync/mysql_to_snowflake.py

```diff
@@ -43,6 +42,18 @@ def partial_sync_table(args: Namespace) -> Union[bool, str]:
         }
 
         snowflake_types = mysql.map_column_types_to_target(args.table)
+
+        # making target table if not exists
+        snowflake.create_table(
+            target_schema=target_schema,
+            table_name=target_table,
+            columns=snowflake_types['columns'],
+            primary_key=snowflake_types.get('primary_key'),
+            is_temporary=False,
+            sort_columns=False,
+            if_not_exists=True
```

Comment:
I find this parameter slightly confusing because it is True on a negative, and also default **False**

Could we change it to be something that is True on a positive, like `table_already_exists` and rework the logic around that ?

Reply:
- [ ] Fix it!